Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -365,6 +365,11 @@ Inspector::Protocol::ErrorStringOr<void> WebAutomationSession::closeBrowsingCont
     if (!page)
         SYNC_FAIL_WITH_PREDEFINED_ERROR(WindowNotFound);
 
+#if PLATFORM(WIN) // TODO: also PLATFORM(PLAYSTATION) should be added
+    // We don't have to close page/window/application, but want to disconnect to automation frontend.
+    terminate();
+#endif
```

Comment:
Why don't we have to close? If you don't have to close, can we put it in `#else`?
~~~
#else
  page->closePage();
#endif
~~~

Reply:
> Why don't we have to close? 

The wincairo WebDriver enabled by this change does not launch a browser when connecting.
WebDriver connects to a browser that is already running and performs automation.
So, I thought it would be better not to close the browser even when disconnecting.

> If you don't have to close, can we put it in #else?

You are right, `closePage()` is not needed and can be placed in #else.